src/components/ElementBar/ColorPickerButton/ColorPicker.test.tsx

```diff
@@ -171,6 +191,85 @@ describe('<ColorPicker/>', () => {
     });
   });
 
+  /**
+   * Renders the colour picker, selects some elements and sets their colours.
+   *
+   * @param elementIds - ID of the elements to set the colour for
+   * @param color - HEX code of the colour to set
+   */
+  const renderColorPickerAndSetElementColors = async (
+    elementIds: string[],
+    color: string,
+  ) => {
+    activeSlide.setActiveElementIds(elementIds);
+    render(<ColorPicker />, { wrapper: Wrapper });
+    await userEvent.click(screen.getByRole('button', { name: 'Pick a color' }));
+    const grid = await screen.findByRole('grid', { name: 'Colors' });
+    await userEvent.click(within(grid).getByRole('button', { name: color }));
+  };
+
+  /**
+   * Extracts a map from element ID to the element colour.
+   *
+   * @param activeSlide - The slide to get the element, color map of
+   * @returns Record that maps element ID to the element colour
+   */
+  const selectElementColors = (
+    activeSlide: WhiteboardSlideInstance,
+  ): Record<string, string> => {
```

Comment:
1. please use `function` keyword here
2. this not needs to be in `describe`, it is independent from any variables and is pure function, please move to the bottom of the file